NitroxClient/GameLogic/Spawning/Metadata/StarshipDoorMetadataProcessor.cs

```diff
@@ -0,0 +1,26 @@
+﻿using NitroxModel.DataStructures.GameLogic.Entities.Metadata;
+using NitroxModel.Logger;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.Spawning.Metadata
+{
+    class StarshipDoorMetadataProcessor : GenericEntityMetadataProcessor<StarshipDoorMetadata>
+    {
+        public override void ProcessMetadata(GameObject gameObject, StarshipDoorMetadata metadata)
+        {
+            Log.Info($"Received door metadata change for {gameObject.name} with data of {metadata}");
```

Comment:
I think that this (and all other lines in the other processors) is more like Debug